python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -243,6 +244,9 @@ def _wrap_image_model(model, examples, model_task, is_function,
     :param number_of_classes: optional parameter specifying the
     number of classes in the dataset
     :type number_of_classes: int
+    :param device: optional parameter specifying the device to move the model
```

Comment:
could we add an auto option as well, and make that the default?  the 'auto' option would just use a device if cuda is available using the old logic.

Reply:
I think that can be done in a separate PR though, this looks good

Reply:
oh I like that idea a lot! Will open a separate PR and merge this one :)